clients/include/testing_spr_strided_batched.hpp

```diff
@@ -79,7 +79,7 @@ void testing_spr_strided_batched(const Arguments& arg)
     }
 
     size_t abs_incx = incx >= 0 ? incx : -incx;
-    size_t size_A   = size_t(N * (N + 1) / 2);
+    size_t size_A   = size_t((N * (N + 1)) / 2);
```

Comment:
```
size_t size_A   = size_t((N * (N + 1)) / 2);
```
Should be:
```
size_t size_A = size_t(N) * (N+1) / 2;
```
i.e., `N` needs to be converted to `size_t` before the multiplication to avoid overflow in case `N` is `rocblas_int`, and the outer parentheses are redundant.